src/system.rs

```diff
@@ -1628,7 +1629,13 @@ macro_rules! system {
     ) => {
         use $path as __system;
 
-        type Units = dyn __system::Units<$V, $($name = __system::$name::$U,)+>;
+        /// Marker trait to identify a [system of units][units] based on the set of given [base
```

Comment:
This is not a trait, is it? Rather a placeholder for an implementor of the trait with concrete base units filled in?

Reply:
Updated the wording slightly. This make more sense?

```rust
        /// `Units` trait type alias to identify a [system of units][units] based on the set of
        /// given [base units][base] of a [system of quantities][quantities].
```